web/src/emoji_picker.js

```diff
@@ -789,10 +789,12 @@ function register_click_handlers() {
         );
     });
 
-    $("body").on("click", "#set-user-status-modal #selected_emoji", (e) => {
+    $("body").on("click keydown", "#set-user-status-modal #selected_emoji", (e) => {
         e.preventDefault();
         e.stopPropagation();
-        toggle_emoji_popover(e.target, undefined, {placement: "bottom"});
+        if (keydown_util.is_enter_event(e) || e.type === "click") {
+            toggle_emoji_popover(e.target, undefined, {placement: "bottom"});
+        }
```

Comment:
I guess the other way we could do this would be to register a `ui_util.convert_enter_to_click` handler for this element; not sure what is better.

Reply:
Your suggested feedback made sense so I have pushed the updated code to reflect the same.